specs/CONTROL.md

```diff
@@ -153,24 +152,24 @@ Response{
 }
 ```
 
-After writing the response message to the socket, the daemon begins piping the
-newly created stream to the client over the socket. Clients may read from and
-write to the socket as if it were the stream. **WARNING**: Clients must be
+After writing the response message to the address, the daemon begins piping the
+newly created stream to the client over the multi-address defined protocol. Clients may read from and
```

Comment:
connection over the multi-address supported transport

Reply:
Done